src/main/org/pantsbuild/jarjar/MainProcessor.java

```diff
@@ -52,7 +54,7 @@ public MainProcessor(List<PatternElement> patterns, boolean verbose, boolean ski
         if (kp != null)
             processors.add(kp);
         processors.add(new ZapProcessor(zapList));
-        processors.add(new JarTransformerChain(new RemappingClassTransformer[]{ new RemappingClassTransformer(pr) }));
```

Comment:
Hmm; yeah that'd probably be better. It would also probably help clean up the option plumbing a bit.